examples/project-service-list-simple/README.md

```diff
@@ -0,0 +1,33 @@
+# project-service-list: Simple Example
+
+### Overview
+
+In this example, we will see how to generate [project services](https://cloud.google.com/config-connector/docs/reference/resource-docs/serviceusage/service) for `compute.googleapis.com` and `redis.googleapis.com`.
+
+### Fetch the example package
+
+Get the example package by running the following commands:
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/project-service-list-simple
+```
+
+### Function invocation
+
+Render the configuration, `project-service-list` has been defined declaratively in the Kptfile and configured via `services.yaml`.
+
+```shell
+$ kpt fn render project-service-list-simple --results-dir /tmp
+```
+
+### Expected result
+
+```shell
+[RUNNING] "gcr.io/kpt-fn/project-service-list:unstable"
+[PASS] "gcr.io/kpt-fn/project-service-list:unstable"
+  Results:
+    [INFO] generated service in object "serviceusage.cnrm.cloud.google.com/v1beta1/Service/proj1-service-compute"
+    [INFO] generated service in object "serviceusage.cnrm.cloud.google.com/v1beta1/Service/proj1-service-redis"
```

Comment:
Note to self: I realized the default out from `kpt fn render` isn't well suited for generating resources. For example if it omitted `in`, the sentence will read better.